homeassistant/components/rainmachine/update.py

```diff
@@ -44,15 +44,14 @@ class UpdateStates(Enum):
 
 UPDATE_DESCRIPTION = RainMachineEntityDescription(
     key="update",
-    name="Firmware",
```

Comment:
@frenck Can you verify that removing device class also works for update entities? I am not 100% sure.